configs/dask_cluster/github_kube_cluster/github_kube_cpu_cluster.py

```diff
@@ -26,7 +26,7 @@
     pod_template="configs/dask_cluster/github_kube_cluster/worker-cpu-github.yml",
     scheduler_pod_template=scheduler_pod_template,
     namespace="zulissi",
-    name="dask-catlas-%s" % os.environ["GITHUB_RUN_ID"],
+    name=f"""dask-catlas-{os.environ["GITHUB_RUN_ID"]}""",
```

Comment:
you can use " on the f str and ' on the inner string. doesnt need to be changed but I think that is a little cleaner